plugins/csi/client.go

```diff
@@ -383,6 +386,109 @@ func (c *client) ControllerValidateCapabilities(ctx context.Context, req *Contro
 	return nil
 }
 
+func (c *client) ControllerCreateVolume(ctx context.Context, req *ControllerCreateVolumeRequest, opts ...grpc.CallOption) (*ControllerCreateVolumeResponse, error) {
+	if c == nil {
+		return nil, fmt.Errorf("Client not initialized")
+	}
+	if c.controllerClient == nil {
+		return nil, fmt.Errorf("controllerClient not initialized")
+	}
```

Comment:
This seems very defensive, and somewhat atypical of other applications.  How likely/realistic is it to run into un-initialized client?

Reply:
Honestly probably never in production... this area of the code has a lot of "make sure this doesn't NPE if I mess it up in development" done back when it was originally written. I was imitating that here but I think you're right there's no point to it. I'll remove what's been added in this PR and do a cleanup PR of the existing code at some point (I think I'd like to split the `plugins/csi` into controller vs node as well instead of client vs plugin, so I'll have some refactoring to do later anyways).